crates/biome_configuration/src/generated/assists.rs

```diff
@@ -0,0 +1,10 @@
+//! Generated file, do not edit by hand, see `xtask/codegen`
+
+use crate::analyzer::assists::*;
+use biome_analyze::{AnalyzerRules, MetadataRegistry};
+pub fn push_to_analyzer_assists(
+    _rules: &Rules,
+    _metadata: &MetadataRegistry,
+    _analyzer_rules: &mut AnalyzerRules,
+) {
+}
```

Comment:
Is it expected that the function be empty?

Reply:
For the time being, yes